orange-datamodel/src/main/java/com/hartwig/hmftools/datamodel/purple/PurpleVariant.java

```diff
@@ -68,4 +66,18 @@ public interface PurpleVariant
 
     @Nullable
     List<Integer> localPhaseSets();
+
+    @Deprecated
+    @Value.Derived
+    default boolean reported()
+    {
+        if(canonicalImpact().reported())
+        {
+            return true;
+        }
+        else
+        {
+            return otherImpacts().stream().anyMatch(PurpleTranscriptImpact::reported);
+        }
+    }
```

Comment:
Do we want this to be deprecated? I thought the idea was to keep this longer term, since otherwise we would have to reimplement the method in multiple data-model users.

Reply:
Also, Orange uses this field itself. If we do want to deprecate it, shouldn't we already remove those calls in this pull request?

Reply:
I'd lean towards deprecation, but no strong opinion. So far the ORANGE datamodel has been as lean-and-mean as possible, meaning there is no room for derived values inside the datamodel. 

And agree on the 2nd item!